operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ControllerConfigurationOverrider.java

```diff
@@ -30,12 +30,12 @@ public class ControllerConfigurationOverrider<R extends HasMetadata> {
   private String fieldManager;
   private Duration reconciliationMaxInterval;
   private Map<DependentResourceSpec, Object> configurations;
-  private final InformerConfigHolder<R>.Builder config;
+  private final InformerConfiguration<R>.Builder config;
 
   private ControllerConfigurationOverrider(ControllerConfiguration<R> original) {
     this.finalizer = original.getFinalizerName();
     this.generationAware = original.isGenerationAware();
-    this.config = InformerConfigHolder.builder(original.getResourceClass())
+    this.config = InformerConfiguration.builder(original.getResourceClass())
```

Comment:
Wouldn't be it confusing that there is now an `InformerEventSourceConfiguration` renamed from `InformerConfiguration`, but there is now a new `InformerConfiguration` for something else?
Also an `@Informer` that is basically the new `InformerConfiguration` just represented by annotation?

Not what would be better, just might be confusing.

Reply:
We would definitely need to document it and add it to the javadoc as well so that the information is available contextually. I think this cleans things nicely this way, though, because having multiple things names InformerConfig-something was getting quite confusing, especially when some of them were not even related to informers at all…